scripts/install_deepchem_conda.sh

```diff
@@ -6,33 +6,33 @@
 export tensorflow=tensorflow
 
 
-if [ -z "$1" ]
-then
-    echo "Must Specify Conda Environment Name"
-fi
-
 if [ -z "$python_version" ]
 then
     echo "Using python 3.5 by default"
     export python_version=3.5
 fi
 
+if [ -z "$1" ];
```

Comment:
Miniconda changes stopped letting us install to "root" environment.  The changed the name to "base", and have lots of issues around it, so I changed the if/then to install it into the current env if none is specified.